Makefile

```
@@ -361,9 +361,19 @@ prepare-manifests:
+# before them to avoid leaving the working tree modified.
+.PHONY: verify-git-tag
+verify-git-tag:
+	@if [[ "$(GIT_TAG)" != v* ]]; then \
+		echo "GIT_TAG must start with v, got \"$(GIT_TAG)\"" >&2; \
+		exit 1; \
+	fi
+
 .PHONY: artifacts
 artifacts: DEST_CHART_DIR="$(ARTIFACTS)"
-artifacts: clean-artifacts kustomize helm-chart-package prepare-manifests ## Generate local artifacts.
+artifacts: verify-git-tag clean-artifacts kustomize helm-chart-package prepare-manifests ## Generate local artifacts.
```

checkmake (0.3.2)

[warning] Target "artifacts" defined multiple times (lines 375 and 376).

(uniquetargets)